thirdweb/modules/nft.py

```diff
@@ -12,28 +12,42 @@
 
 
 class NftModule(BaseModule):
-    """ 
+    """
     NFT Methods
     """
 
     address: str
     __abi_module: NFT
 
+
     def __init__(self, address: str, client: Web3):
+        """
+        Initializing the class attributes
+        """
         super().__init__()
         self.address = address
+        """
+        NFT module contract address [https://thirdweb.com/: Dashboard: Project ➝ NFT Module]
+        """
+
         self.__abi_module = NFT(client, address)
+        """
+        The ABI makes calls the EVM. Client is default 'Web3' and Address is nft module contract address.
+        """
 
     def mint(self, arg: MintArg) -> NftType:
         """
-        Mints a new token
+        - Mints a new token. 
+        - Arguments passed: Note, a class is used -> MintArg(name, description, image_uri, properties) *Preferrably, using a link
+        - Returns the 'NftMetadata(name,description,image,properties,id,uri)
```

Comment:
this should either be \`NftMetadata(name,description,image,properties,id,uri)\` or NftMetadata(name,description,image,properties,id,uri)